forgotpassword/js/passwordrecovery.js

```diff
@@ -32,7 +31,7 @@ $(document).ready(function()
         $('#emailfield').removeClass();
     }
 
-    function checkEmpty(){
+    function checkEmpty() {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*